Dtos/Users/GetUserDto.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace webapi.Dtos.Users
```

Comment:
make one general dto name as UsersDTO must have all properties in it.

Reply:
Done